### DISCUSSION TITLE
what is bootstrap used for?

### DISCUSSION THREAD

**Discussion by marcelloh** [confetti-framework/confetti#117] [Q&A]
Are people manipulating this?

I can't find any reason (yet) to do so.

Perhaps if nobody does this, it should be part of the foundation.

**Comment by reindert-vetter** [comment]
bootstrap/app.go is present because https://github.com/confetti-framework/confetti/blob/d15b3955691d1808a7159192c87a5b61380d2465/app/http/decorator/register_providers.go#L12 must be able to fetch the providers. Now it cannot move to foundation because they must be able to pick up the providers. maybe I can pass the providers in a different way. I'll look into that.
